private/pkg/manifest/manifest.go

```diff
@@ -139,6 +140,18 @@ func (m *Manifest) Paths() []string {
 	return paths
 }
 
+// Range invokes a function for all the paths in the manifest, passing the path
+// and its digest. The order in which the paths are iterated is not guaranteed.
+// This func will stop iterating if an error is returned.
+func (m *Manifest) Range(f func(path string, digest Digest) error) error {
```

Comment:
Renaming func to follow suggestion in [this comment](https://github.com/bufbuild/buf/pull/1724#discussion_r1069914158). cc @jhump